primitives/npos-elections/compact/src/lib.rs

```diff
@@ -256,6 +258,29 @@ fn struct_def(
 				Ok(assignments)
 			}
 		}
+		type __IndexAssignment = _npos::IndexAssignment<
```

Comment:
You can make a reusable 

```
/// A type alias for [`IndexAssignment`] made from [`crate::CompactSolution`].
pub type IndexAssignmentOf<C> = IndexAssignment<
	<C as crate::CompactSolution>::Voter,
	<C as crate::CompactSolution>::Target,
	<C as crate::CompactSolution>::Accuracy,
>
``` 

in `assignment.rs` for this, IMO.

Reply:
3dda173e4